app/src/main/java/com/kickstarter/viewmodels/ProjectViewModel.kt

```diff
@@ -796,6 +809,10 @@ interface ProjectViewModel {
             this.contactCreatorClicked.onNext(null)
         }
 
+        override fun creatorDashboardButtonClicked() {
```

Comment:
let's rename all of these inputs to match the delegate methods

Reply:
So, I looked into it 🔍 it's moving into refactor territory and Out of Scope™ for this PR

Reply:
Oh yeah... let's save it for a future Investment Day™️